### PR TITLE
DBZ-8121 Document pgvector

### PR DESCRIPTION
**Pull request by jpechane**
Documents https://github.com/debezium/debezium/pull/5735

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 1]
```diff
@@ -1912,6 +1912,37 @@ For format details, see http://www.opengeospatial.org/standards/sfa[Open Geospat
 
 |===
 
+[id="postgresql-pgvector-types"]
+=== PgVector types
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1916]:
```suggestion
=== pgvector types
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 2]
```diff
@@ -1912,6 +1912,37 @@ For format details, see http://www.opengeospatial.org/standards/sfa[Open Geospat
...
+
+The PostgreSQL connector supports all link:https://github.com/pgvector/pgvector[pgvector extension data types].
+
+.Mappings of pgvector data types
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1920]:
```suggestion
.Mappings of PostgreSQL `pgvector` data types
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 3]
```diff
@@ -1912,6 +1912,37 @@ For format details, see http://www.opengeospatial.org/standards/sfa[Open Geospat
...
+|`io.debezium.data.HalfVector` +
+
+|`SPARSEVEC`
+|`STRUCT`
+|`io.debezium.data.SparseVector` +
+ +
+Contains a structure with two fields: +
+
+* `dimensions (INT16)` - Dimensions (total length of the sparse vector) of the sparse vector.
+* `vector (MAP (INT16, FLOAT64))` - A map representing the sparse vector. Each value is is a combination of index of vector element (starting with `1`) and the value of the element.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1942]:
```suggestion
Contains a structure that includes the following fields: 

`dimensions (INT16)`:: The total length of the sparse vector.
`vector (MAP (INT16, FLOAT64))`:: A map that represents the sparse vector.  +
Each map value includes the following elements:

* Index number of the vector element (starting with `1`).
* Value of the vector element.
```

File: documentation/modules/ROOT/pages/connectors/postgresql.adoc [hunk 4]
```diff
@@ -1912,6 +1912,37 @@ For format details, see http://www.opengeospatial.org/standards/sfa[Open Geospat
...
+|`io.debezium.data.Vector` +
+
+|`HALFVEC`
+|`ARRAY (FLOAT32)`
+|`io.debezium.data.HalfVector` +
+
+|`SPARSEVEC`
+|`STRUCT`
+|`io.debezium.data.SparseVector` +
+ +
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/postgresql.adoc:1938]:
```suggestion
|`io.debezium.data.SparseVector` 
 
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
Restructured the Notes description. Does this work?

**Comment by jpechane** [comment]
@roldanbob Thanks for the review!

**Pull request merged**